go/vt/wrangler/shard.go

```diff
@@ -81,26 +74,20 @@ func (wr *Wrangler) SetShardServedTypes(ctx context.Context, keyspace, shard str
 	defer unlock(&err)
 
 	// and update the shard
-	_, err = wr.ts.UpdateShardFields(ctx, keyspace, shard, func(si *topo.ShardInfo) error {
-		return si.UpdateServedTypesMap(servedType, cells, remove)
-	})
+	// TODO: What should we do with this method?
```

Comment:
Seems like `SetShardServedTypes()` should probably be deleted along with the vtctl command.

If not, this method probably at least raise an error